packages/lit-element/src/lit-element.ts

```diff
@@ -141,6 +141,20 @@ export class LitElement extends ReactiveElement {
   }
 }
 
+/**
+ * Constructor type for easier definition of mixins that apply to LitElement.
+ *
+ * ```ts
+ * const MyMixin = <T extends LitElementConstructor>(superClass: T) => {
+ *   class MyMixinSubclass extends superClass {
+ *     // ...
+ *   }
+ *   return MyMixinSubclass;
+ * }
+ * ```
+ */
+export type LitElementConstructor = new (...args: unknown[]) => LitElement;
```

Comment:
Did you try this in an actual mixin? I thought it had to be `new (...args: any[])`